README.md

```diff
@@ -1,4 +1,8 @@
 # What is UTBotCpp?
+
+[![Build UTBot and run unit tests](https://github.com/UnitTestBot/UTBotCpp/actions/workflows/build-utbot.yml/badge.svg)](https://github.com/UnitTestBot/UTBotCpp/actions/workflows/build-utbot.yml)
+[![Publish UTBot as an archive](https://github.com/UnitTestBot/UTBotCpp/actions/workflows/publish-utbot.yml/badge.svg)](https://github.com/UnitTestBot/UTBotCpp/actions/workflows/publish-utbot.yml)
+
```

Comment:
Is it related to the issue? If not, please move to a separate commit or PR

Reply:
Delete from commit